ProteoformSuiteGUI/LoadResults.Designer.cs

```diff
@@ -36,22 +36,22 @@ private void InitializeComponent()
             this.rb_standardOptions = new System.Windows.Forms.RadioButton();
             this.rb_chemicalCalibration = new System.Windows.Forms.RadioButton();
             this.box_parameters = new System.Windows.Forms.GroupBox();
+            this.param_splitcontainer = new System.Windows.Forms.SplitContainer();
             this.calib_stand_splitContainer = new System.Windows.Forms.SplitContainer();
             this.rb_neucode = new System.Windows.Forms.RadioButton();
             this.rb_unlabeled = new System.Windows.Forms.RadioButton();
+            this.label_mincharge = new System.Windows.Forms.Label();
+            this.nud_mincharge = new System.Windows.Forms.NumericUpDown();
             this.cb_calibrate_td_files = new System.Windows.Forms.CheckBox();
             this.cb_calibrate_raw_files = new System.Windows.Forms.CheckBox();
-            this.param_splitcontainer = new System.Windows.Forms.SplitContainer();
-            this.label_minRT = new System.Windows.Forms.Label();
             this.label_maxRT = new System.Windows.Forms.Label();
-            this.label_mincharge = new System.Windows.Forms.Label();
-            this.label_maxcharge = new System.Windows.Forms.Label();
-            this.nud_mincharge = new System.Windows.Forms.NumericUpDown();
-            this.nud_maxcharge = new System.Windows.Forms.NumericUpDown();
+            this.label_minRT = new System.Windows.Forms.Label();
```

Comment:
Are these changes intentional, or are they reverting the changes made to the LoadResults GUI from earlier this year?

Reply:
Sorry I didn't intend to change any part of the ProteoforSuiteGUI. I'll go clean up the unwanted changes tomorrow!

Reply:
Got it! Thanks for the note. I reverted these changes to the Windows Forms GUI. I'll commit those after merging this.